rocketpy/motors/HybridMotor.py

```diff
@@ -493,6 +493,7 @@ def addTank(self, tank, position):
         """
         self.liquid.addTank(tank, position)
         self.solid.massFlowRate = self.totalMassFlowRate - self.liquid.massFlowRate
+        reset_funcified_methods(self)
```

Comment:
Really clean. Perhaps in the future we can deal with this caching in another way, but this reset really helps to reset the caching when needed and to not miss any funcify by mistake.